reactos/dll/win32/browseui/travellog.cpp

```diff
@@ -172,6 +172,7 @@ HRESULT STDMETHODCALLTYPE CTravelEntry::Update(IUnknown *punk, BOOL fIsLocalAnch
     GetToolTipText(punk, wch);
     TRACE("Updating entry with display name: %S\n", wch);
 
+    ZeroMemory(&windowData, sizeof(WINDOWDATA));
```

Comment:
I was talking about MS mshtml code (when trying to load a webpage on Windows with our browser), not ours (which works as expected).
I may capture a backtrace maybe this evening.